lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -296,6 +398,9 @@ def main():
             organizationalUnitName=dict(aliases=['OU'], type='str'),
             commonName=dict(aliases=['CN'], type='str'),
             emailAddress=dict(aliases=['E'], type='str'),
+            subjectAltName=dict(aliases=['subjectAltName'], type='list'),
```

Comment:
Not your fault, but can you remove the aliases[] please, since its the same value